### PR TITLE
hover effect on navbar links

### LINKED ISSUE #494: Apply hover effect on navbar links
Screenshot 2021-08-05 114553

### PR DESCRIPTION
**Pull request by shreyshreyansh**
#### Fixes #494 

Screenshot 2021-08-05 115029

### CODE REVIEW

File: static/style.css [hunk 1]
```diff
@@ -92,18 +92,22 @@ section div:nth-child(6) img {
   color: white;
   font-size: 20px;
   margin-left: 20px;
-
+  border-radius: 10px;
 }
 
+#header-nav .nav-link:hover {
```
Comment (shreyshreyansh) [inline static/style.css:98]: adding a hover effect

**Comment by QAZIMAAZARSHAD** [comment]
Nice but implement this on every single navbar.

Check. 

And don't disturb any other feature,

When you are done share a live link so I can review your changes.

**Comment by shreyshreyansh** [comment]
It is working for every links on the navbar.. it is just a photo

**Comment by QAZIMAAZARSHAD** [comment]
> It is working for every links on the navbar.. it is just a photo

Share website link.

**Pull request closed without merging**